spec/rspec/core/did_you_mean_spec.rb

```diff
@@ -0,0 +1,42 @@
+module RSpec
+  module Core
+    RSpec.describe DidYouMean do
+      describe '#call' do
+        if defined?(::DidYouMean::SpellChecker)
+          context "when `DidYouMean::SpellChecker` is available", skip: !defined?(::DidYouMean::SpellChecker) do
```

Comment:
For 1.8.7 etc this needs to be `:skip =>`

Reply:
Of course. Done